spec/features/users/google_oauth_spec.rb

```diff
@@ -6,9 +6,9 @@
       {"provider"=>"google_oauth2",
       "uid"=>"110287008629109688070",
       "info"=>
-      {"name"=>"gardenthat",
-      "email"=>"gardenthat@gmail.com",
-      "unverified_email"=>"gardenthat@gmail.com",
+      {"name"=>"gardenthattesting",
+      "email"=>"gardenthattesting@gmail.com",
+      "unverified_email"=>"gardenthattesting@gmail.com",
```

Comment:
changes the test user and test user credentials (the new TEST_USER_GOOGLE_TOKEN and TEST_USER_GOOGLE_REFRESH_TOKEN with `calendar` scope will be sent separately)

Reply:
Added to my .yml folder. thanks for catching this.